python-phoenixdb/phoenixdb/__init__.py

```diff
@@ -56,7 +56,7 @@
 
 
 def connect(url, max_retries=None, auth=None, authentication=None, avatica_user=None, avatica_password=None,
-            truststore=None, verify=None, do_as=None, user=None, password=None, **kwargs):
+            truststore=None, verify=None, do_as=None, user=None, password=None, headers=None, **kwargs):
```

Comment:
Better call this extra_headers.
Also, add a description for the new parameter

Reply:
Done